torchtune/datasets/_raft.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from torchtune.data import RAFTInstructTemplate
+from torchtune.datasets._instruct_raft import InstructDatasetDeepLakeRAFT
+from torchtune.modules.tokenizers import Tokenizer
+
+activeloop_dataset = "hub://manufe/raft_format_dataset_biomedical"  # Replace with your ActiveLoop dataset
+
+
+def raft_dataset(
```

Comment:
What is this dataset exactly? Seems like `raft_dataset` is still the generic dataset type. But for dataset builders like these we typically point to a canonical dataset on the Hub (e.g. Alpaca). Is `raft_format_dataset_biomedical` the canonical dataset here? If so, is this something that's widely-used, or intended more as a demo?

Reply:
RAFT being a fairly new technique there is no real reference dataset, this is a dataset that we are using on a real project and can be used as a demo

Reply:
here too perhaps we can replace the dataset with an input taken in the yaml file or passed as a parameter in the cli